src/exe/udp_client_test/CMakeLists.txt

```diff
@@ -0,0 +1,9 @@
+# TODO Find out why this isn't building the target
```

Comment:
Callout - not sure what I'm missing here. 

To test locally, you can still `cd` into the build directory and manually run `make`.